security_check/main.py

```diff
@@ -51,7 +51,10 @@ def _find_base_image(image):
     img = parsed.get('image_analysis')
     if img:
         base_img_url = img[0]['base_image_url']
-        return base_img_url[len('https://'):base_img_url.find('@')]
+        base_img = base_img_url[len('https://'):base_img_url.find('@')]
```

Comment:
Could you make this a tiny bit more generic, and maybe have a map of substitutions?